Authentication/Login/View/LoginViewConfiguration.swift

```diff
@@ -6,8 +6,12 @@
 //  Copyright © 2016 Wolox. All rights reserved.
 //
 
-import Foundation
-
+/*
+    Represents the configurations able to
```

Comment:
What about `Represents the configurable parameters of a view that conforms to LoginViewType`?